docs/CONTRIBUTING.md

```diff
@@ -46,6 +46,13 @@ direnv: export +AR +AS +CC +CONFIG_SHELL +CXX +HOST_PATH +IN_NIX_SHELL +LD +NIX_
 🎉
 ```
 
+> **Note:** On macOS, a [direnv bug](https://github.com/direnv/direnv/issues/1345) can cause `nix-shell` to fail to build or run `coder`. If you see `error: creating directory` when attempting to run, build, or test, adding one line to your `.envrc` should fix the problem:
```

Comment:
Honestly, I would create a subsection for Troubleshooting and mention this issue. Otherwise, there is a risk we'll end up with 10 notes referring different issues.

I'll defer the final decision to @EdwardAngert 👍

Reply:
thanks @paulroub ! I do agree with @mtojek , but there are a few more (unrelated) things I want to edit in this doc. I'll approve this as is, and follow-up in a little while

Reply:
Is there a trick (beyond commenting "recheck", which isn't doing it) to have the `contrib / cla` check redone?

Reply:
It might just be busted, @paulroub 

I had this issue with some of my own PRs - some I was able to fix when I confirmed my git config user and email (pasting below, just in case), and some just still fail. We're working on figuring out why and fixing it, but it's not a required check for us to merge

```shell
git config --local user.name "myusername" && \
git config --local user.email "no-reply-email+myusername@users.noreply.github.com"
```